beat/redisbeat.go

```diff
@@ -156,27 +161,28 @@ func (rb *Redisbeat) Config(b *beat.Beat) error {
 	}
 
 	logp.Debug("redisbeat", "Init redisbeat")
-	logp.Debug("redisbeat", "Period %v\n", rb.period)
-	logp.Debug("redisbeat", "Host %v\n", rb.host)
-	logp.Debug("redisbeat", "Port %v\n", rb.port)
-	logp.Debug("redisbeat", "Network %v\n", rb.network)
-	logp.Debug("redisbeat", "Max Connections %v\n", rb.maxConn)
-	logp.Debug("redisbeat", "Auth %t\n", rb.auth)
-	logp.Debug("redisbeat", "Server statistics %t\n", rb.serverStats)
-	logp.Debug("redisbeat", "Client statistics %t\n", rb.clientsStats)
-	logp.Debug("redisbeat", "Memory statistics %t\n", rb.memoryStats)
-	logp.Debug("redisbeat", "Persistence statistics %t\n", rb.persistenceStats)
-	logp.Debug("redisbeat", "Stats statistics %t\n", rb.statsStats)
-	logp.Debug("redisbeat", "Replication statistics %t\n", rb.replicationStats)
-	logp.Debug("redisbeat", "Cpu statistics %t\n", rb.cpuStats)
-	logp.Debug("redisbeat", "Command statistics %t\n", rb.commandStats)
-	logp.Debug("redisbeat", "Cluster statistics %t\n", rb.clusterStats)
-	logp.Debug("redisbeat", "Keyspace statistics %t\n", rb.keyspaceStats)
+	logp.Debug("redisbeat", "Period %v", rb.period)
+	logp.Debug("redisbeat", "Host %v", rb.host)
+	logp.Debug("redisbeat", "Port %v", rb.port)
+	logp.Debug("redisbeat", "Network %v", rb.network)
+	logp.Debug("redisbeat", "Max Connections %v", rb.maxConn)
+	logp.Debug("redisbeat", "Auth %t", rb.auth)
+	logp.Debug("redisbeat", "Server statistics %t", rb.serverStats)
+	logp.Debug("redisbeat", "Client statistics %t", rb.clientsStats)
+	logp.Debug("redisbeat", "Memory statistics %t", rb.memoryStats)
+	logp.Debug("redisbeat", "Persistence statistics %t", rb.persistenceStats)
+	logp.Debug("redisbeat", "Stats statistics %t", rb.statsStats)
+	logp.Debug("redisbeat", "Replication statistics %t", rb.replicationStats)
+	logp.Debug("redisbeat", "Cpu statistics %t", rb.cpuStats)
+	logp.Debug("redisbeat", "Command statistics %t", rb.commandStats)
+	logp.Debug("redisbeat", "Cluster statistics %t", rb.clusterStats)
+	logp.Debug("redisbeat", "Keyspace statistics %t", rb.keyspaceStats)
+	logp.Debug("redisbeat", "RequiredPass %v", rb.pass)
```

Comment:
I'd rather not echo back the password, even though it's stored in a config file somewhere.